File: back_old/src/main/kotlin/ru/lanit/rnd/files/config/NettyWebServerFactoryPortCustomizer.java
package ru.lanit.rnd.files.config;

import org.springframework.boot.web.embedded.netty.NettyReactiveWebServerFactory;
import org.springframework.boot.web.server.WebServerFactoryCustomizer;
import org.springframework.stereotype.Component;

@Component
public class NettyWebServerFactoryPortCustomizer
        implements WebServerFactoryCustomizer<NettyReactiveWebServerFactory> {

    @Override
    public void customize(NettyReactiveWebServerFactory serverFactory) {
        serverFactory.setPort(8080);
    }
}

File: back/src/main/kotlin/ru/lanit/rnd/files/config/NettyWebServerFactoryPortCustomizer.kt
package ru.lanit.rnd.files.config;

import org.springframework.boot.web.embedded.netty.NettyReactiveWebServerFactory;
import org.springframework.boot.web.server.WebServerFactoryCustomizer;
import org.springframework.stereotype.Component;

@Component
class NettyWebServerFactoryPortCustomizer : WebServerFactoryCustomizer<NettyReactiveWebServerFactory> {

    override fun customize(serverFactory: NettyReactiveWebServerFactory) {
        serverFactory.setPort(8080);
    }
}